src/dag-node.js

```diff
@@ -52,7 +53,7 @@ function DAGNode (data, links) {
   this.addRawLink = (link) => {
     encoded = null
     this.links.push(new DAGLink(link.name, link.size, link.hash))
-    this.links.sort(linkSort)
+    stable(this.links,linkSort)
```

Comment:
From the doc, it appears that `stable` doesn't do in-place sorting by default, unlike the built-in `Array#sort`. You have to use `stable.inplace`.